src/components/Dashboard/Dashboard.js

```diff
@@ -1,17 +1,25 @@
 import React from "react";
+import { Route, withRouter, Switch } from "react-router-dom";
 import styles from "./Dashboard.scss";
-import ProgressBar from "./ProgressBar/ProgressBar";
-import FormSteps from "./Form/FormSteps";
+import routes from "../../routing/routes";
 
 const Dashboard = () => {
   return (
     <div className={styles.Dashboard}>
       <div className={styles.Wrapper}>
-        <ProgressBar />
-        <FormSteps />
+        <Switch>
+          {Object.keys(routes).map(key => (
```

Comment:
Do we have a plan to use these keys somewhere? Maybe routes could just be an ordinary array instead of an object. It would make it a little bit clearer, since we don't use it anywhere